app/helpers/assignment_index_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AssignmentIndexHelper
+  def cal_day(day:)
+    'current-day' if day == Date.current
+  end
+
+  def cal_assignment(assignment:, day:)
+    return if assignment.blank?
+    ass = 'assignment'
```

Comment:
@jforgue, what was the first thing to do on your list? (Hint: it has to do with abbreviations)